src/crypto/crypto_util.cc

```diff
@@ -136,7 +136,13 @@ bool InitCryptoOnce(Isolate* isolate) {
   return true;
 }
 
+// Protect accesses to FIPS state with a mutex. This should potentially
+// be part of a larger mutex for global OpenSSL state.
+static Mutex fips_mutex;
```

Comment:
This makes sense for semantic reasons, but is my understanding correct that this is technically not required since `per_process::cli_options_mutex` will already guarantee mutually exclusive access whenever `fips_mutex` is used?

Reply:
Yes, that is correct as far as the current state of this PR is concerned. :+1:

Reply:
Thanks for clarifying! 😃